src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isPerson = sex === 'm';
```

Comment:
The variable name is a little unclear, you can add it like this
```suggestion
  const partner= sex === 'm' ? 'wife' : 'husband';
```